src/partials/layout/_footer.html

```diff
@@ -0,0 +1,14 @@
+<footer class="footer">
+  <div class="container">
+    <div class="footer-wrapper">
```

Comment:
You have container div already, why do you need wrapper div in it?

Reply:
I removed the div container. Not needed for positioning.